internal/unleash/unleash.go

```diff
@@ -0,0 +1,38 @@
+package unleash
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/Unleash/unleash-client-go/v4"
+	ucontext "github.com/Unleash/unleash-client-go/v4/context"
+)
+
+type FeatureFlag string
+
+const (
+	CompliancePolicies FeatureFlag = "image-builder.compliance-policies.enabled"
+)
+
+type Config struct {
+	URL   string
+	Token string
+}
+
+func Initialize(conf Config) error {
+	err := unleash.Initialize(
+		unleash.WithListener(LogListener{}),
+		unleash.WithUrl(conf.URL),
+		unleash.WithCustomHeaders(http.Header{"Authorization": {conf.Token}}),
+	)
+	if err != nil {
+		return fmt.Errorf("Unleash error: %w", err)
+	}
+
+	return nil
+}
+
+func Enabled(flag FeatureFlag) bool {
+	ctx := ucontext.Context{}
+	return unleash.IsEnabled(string(flag), unleash.WithContext(ctx), unleash.WithFallback(true))
+}
```

Comment:
What we find very useful is to also send org id to unleash so you can turn on features per accounts.

Reply:
Hmm interesting, how? Do you add it to the context somehow?

Reply:
There are several options, edge for example uses the native "OrgId" field from the [Unleash Context](https://github.com/Unleash/unleash-client-go?tab=readme-ov-file#unleash-context) to pass this information. That is a reasonable approach.

Reply:
Neat neat, mind if we do it in a followup? Definitely interesting.